### DISCUSSION TITLE
Livewire pagination url path is change when using search

### DISCUSSION THREAD

**Discussion by oka-wiadnyana** [livewire/livewire#7073] [1. Help]
I'm facing really weird condition with live wire pagination. It's working normally when access the page in the first time..but when I use search condition...the page button set a strange url which is http://localhost/repo/public/livewire/update?page=2. The url should be http://localhost/repo/public/admin/proccess?page=2. Please help me..how to fix it?
I already search in tje discussion..but i can't find the solution..

**Comment by malogajski** [comment]
If you're utilizing custom pagination, ensure that you've opted for `<button>` tags instead of `<a>` tags. If you've modified the file obtained through the `php artisan livewire:publish --pagination` command, you can refer to the view in the vendor folder for guidance: `vendor/livewire/livewire/src/Features/SupportPagination/views/tailwind.blade.php`.
